test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -21,6 +24,15 @@
   ([resp keywordize?]
    (-> resp :body slurp (json/parse-string keywordize?))))
 
+(defn jolokia-encode
```

Comment:
Cool. I think @camlow325 was thinking it would be good to add a DOCs ticket for the /metrics/v2 endpoint, so we can just add a comment about the escaping to that pointing at the escape rules docs.